kubernetes-client-api/src/main/java/io/fabric8/kubernetes/client/utils/KubernetesResourceUtil.java

```diff
@@ -50,6 +50,7 @@ private KubernetesResourceUtil() {
 
   public static final Pattern KUBERNETES_DNS1123_LABEL_REGEX = Pattern.compile("[a-z0-9]([-a-z0-9]{0,61}[a-z0-9])?");
   private static final Pattern INVALID_LABEL_CHARS_PATTERN = Pattern.compile("[^-A-Za-z0-9]+");
+  private static final String DEFAULT_DOCKER_REGISTRY_SECRET_NAME = "docker-registry-secret";
```

Comment:
Minor: I wouldn't use the word `docker` in the name as it is just one possible registry. What about `container-image-registry-secret` ?

Reply:
Thank you for your good suggestion!

I removed the duplication and use more general name as you suggest!

Reply:
@sunix
Thank you for your good suggestion!

I removed the duplication and used more general name as you suggest